src/common.tex

```diff
@@ -21,8 +19,9 @@
 \tolerance=2000 
 \emergencystretch=10pt
 
-% REFERENCE
-\def\REFERENCE#1{}
+% Define REFERENCE and OPTIONAL commands to handle citations
+\newcommand{\REFERENCE}[1]{} % Empty command that takes one argument
+\newcommand{\OPTIONAL}[1]{} % Empty command that takes one argument
```

Comment:
2. these commands essentially do nothing

Reply:
Yeah that's the idea, just references.  I forget what \OPTIONAL was for now.